tests/library.dylan

```diff
@@ -0,0 +1,21 @@
+Module: dylan-user
+
+define library anaphora-test
+  use common-dylan;
+  use testworks;
+  use anaphora;
+
+  export anaphora-tests;
+end library anaphora-test;
+
+define module anaphora-test
+  use common-dylan;
+  use common-extensions,
+    import: { integer-to-string };
```

Comment:
Everything in `common-extensions` is reexported from `common-dylan` so you shouldn't need this separate import.

Reply:
Thank you. I'll change it.